src/scheduler.cc

```diff
@@ -397,37 +396,27 @@ bool SchedulerService::can_run(const Task& task) {
 
 std::pair<WorkerId, ObjStoreId> SchedulerService::register_worker(const std::string& worker_address, const std::string& objstore_address, bool is_driver) {
   RAY_LOG(RAY_INFO, "registering worker " << worker_address << " connected to object store " << objstore_address);
-  ObjStoreId objstoreid = std::numeric_limits<size_t>::max();
+  auto result = std::make_pair(std::numeric_limits<WorkerId>::max(), std::numeric_limits<ObjStoreId>::max());
```

Comment:
not a super big deal, but I find it clearer to use the variable names `objstoreid` and `workerid` instead of `result.first` and `result.second`, and then just construct the pair at the end of the method.